action.yml

```diff
@@ -4,14 +4,19 @@ author: "chrnorm"
 inputs:
   state:
     description: "State for the deployment"
+    required: true
```

Comment:
Not sure, can you double-check, @chrnorm?

Reply:
Looks good to me @localheinz, thanks for the PR!